src/text-area/index.ts

```diff
@@ -0,0 +1,14 @@
+import { customElement } from "lit/decorators.js";
+import { TextArea } from "./text-area";
+import styles from "./text-area.style";
+
+@customElement("tap-text-area")
```

Comment:
@Negarnbkhti42 element name should be `tap-textarea` to match the native `textarea`